elasticsearch5/docker-compose.yml

```diff
@@ -20,10 +20,33 @@ elasticsearch:
     - elasticsearchshareddata
 
 elasticsearchshareddata:
-  image: registry.service.opg.digital/opguk/elasticsearch-shared-data:latest
+  image: registry.service.opg.digital/opguk/elasticsearch-shared-data
+
+elasticsearchingest:
+  image: registry.service.opg.digital/opguk/elasticsearch5
+  log_driver: syslog
+  mem_limit: 5000000000
+  links:
+    - elasticsearch
+  ports:
+    - 9201:9200
+  env_file: ./ingestenv
+  environment:
+    ELASTICSEARCH_NODE_TYPE: 'ingest'
+
+
+#elasticsearchcoord:
+#  image: registry.service.opg.digital/opguk/elasticsearch5
+#  log_driver: syslog
+#  mem_limit: 5000000000
+#  ports:
+#    - 9200:9200
+#  env_file: ./coordenv
+#  environment:
+#       ELASTICSEARCH_NODE_TYPE: 'coord'
 
 #elasticcurator:
-#  image: {{ registry_url }}/opguk/elasticsearch5:{{es_version}}
+#  image: registry.service.opg.digital/opguk/elasticsearch5:{{es_version}}
```

Comment:
the registry url value is a pillar value, so is not relevant. this compose.yml is for testing, not deployment